Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -230,12 +230,10 @@ class BocList
       return false;
     }  
   
-    var currentRowBlock = new BocList_RowBlock (currentRow, selectorControl);
-    var selectedRows = BocList._selectedRows[bocList.id]!;
-    var isRowHighlightingEnabled = true
-    var isCtrlKeyPress = false;
-    if (evt)
-      isCtrlKeyPress = evt.ctrlKey;
+    const currentRowBlock = new BocList_RowBlock (currentRow, selectorControl);
+    const selectedRows = BocList._selectedRows[bocList.id]!;
+    const isRowHighlightingEnabled = true
+    const isCtrlKeyPress = !!evt?.ctrlKey;
```

Comment:
can we also do this?
```suggestion
    const isCtrlKeyPress = evt?.ctrlKey == true;
```
(either with `==` or `===`). The leading double not is not readable.